x/batch/decorator_test.go

```diff
@@ -32,31 +34,55 @@ func (wrongWeaveMsg) Path() string {
 	panic("implement me")
 }
 
-type mockHelper struct {
-	mock.Mock
+type txMock struct {
```

Comment:
You can use [`weavetest.Tx`](https://github.com/iov-one/weave/blob/master/weavetest/tx.go#L12) instead as it provides the same functionality.